### DISCUSSION TITLE
PP-OCRv4或其他PP-OCR系列预训练包含这个数据集（HW_Chinese）了吗？

### DISCUSSION THREAD

**Discussion by IceHowe** [PaddlePaddle/PaddleOCR#13485] [Q&A]
请问PP-OCRv4或其他PP-OCR系列提供的预训练权重使用这个数据集（HW_Chinese）训练过了吗？
https://aistudio.baidu.com/datasetdetail/102884/0
![image](https://github.com/user-attachments/assets/7ceab1d3-cd20-4d78-b353-a1199de21bfd)

**Comment by UserWangZz** [comment]
你好，pp-ocr系列模型没有使用这个数据集进行训练

**Reply by IceHowe** [reply]
你好，可以问一下PP-OCR系列或者提供的其他模型的权重是在什么数据集训练的吗？

**Reply by UserWangZz** [reply]
不好意思，是内部数据集，不公开的，你可以使用公开数据集+自己的场景微调模型试试

**Reply by IceHowe** [reply]
> 你好，pp-ocr系列模型没有使用这个数据集进行训练

好的，不过我只是想问问是哪些数据集，看看是不是网上公开的那些，以防我重复训练了，谢谢
